portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,6 +41,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("text-input");
+
+    // Add comment to datastore as new entity.
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity commentEntity = new Entity("Comment");
```

Comment:
"Comment" is the unique key for this entity, right? Would we want to store them to different IDs later on? 

It's fine that the question isn't answered now. But let's put a TODO if we plan to update this in the future.

Reply:
Thank you Bill! I will put it as a TODO rn and explore adding unique IDs when I get around to drawing data from datastore.